spec/unit/models/project_spec.rb

```diff
@@ -40,4 +40,51 @@
       end
     end
   end
+
+  describe '#blueprint' do
+    let(:project_data) do
+      { 'project' => {
+          'links' => { 'self' => '/gdc/foo' },
```

Comment:
Style/IndentHash: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.